news/4615.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix bug where environment wouldn't activate in paths containing () and [] symbols - Related to #4538
```

Comment:
Since #4538 is already listed in the changelog, it does not need to be included here.

Reply:
Got it, thanks. It's updated in the latest push.